packages/@aws-cdk/aws-cloudwatch/lib/alarm-action.ts

```diff
@@ -0,0 +1,19 @@
+import { Construct } from "@aws-cdk/cdk";
+import { IAlarm } from "./alarm";
+
+/**
+ * Interface for objects that can be the targets of CloudWatch alarm actions
+ */
+export interface IAlarmAction {
+  bind(scope: Construct, alarm: IAlarm): AlarmActionProperties;
```

Comment:
The name of the return type is not super consistent across integrations, ha? Should we standardize?

Reply:
It should be `Properties`. I didn't want to call it `Props` because it's not a "Props" object as such, but I may have failed to rename in one PR somewhere.

Another potential candidate is `Attributes`.

Your preference?